ansible/playbooks/roles/opendistro_for_elasticsearch/tasks/log4j-patch.yml

```diff
@@ -0,0 +1,68 @@
+
+- name: Log4j patch
```

Comment:
Maybe lets add the specific CEV nr in the name for future generations so we know why we patch this?

Reply:
Ok so as discussed during standup, it fixes a whole bunch of issues, so lets leave as is.